.travis.yml

```
@@ -99,6 +100,16 @@ jobs:
     env:
       TARGET_PKG: cherry_picker
 
+  - os: windows
+    language: sh
+    python: 3.7
+    before_install:
+      - choco install python --version 3.7
+      - export PATH="/c/Python37:/c/Python37/Scripts:$PATH"
+      - python -m pip install --upgrade pip wheel
+      - *install-flit
+    <<: *install-and-test-cherry-picker
```

Can we put env var declaration into a separate section, please?
```suggestion
    <<: *install-and-test-cherry-picker
    env:
      PATH: >-
        /c/Python37:/c/Python37/Scripts:$PATH
      TARGET_PKG: cherry_picker
```